### PR TITLE
 y/index.md

### PR DESCRIPTION
**Pull request by moro-patana**
Hey,
We created an index.md in 'Y'. Please have a look and check if there are some mistakes.

Thank you,

Katimjk

### CODE REVIEW

File: y/index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+List of artists starting with Y
+
+- [Yoann Loic](yoann-loic.md)
+
+Go back to the artist index
```
Comment (ndorfin) [inline y/index.md]: This text needs to be a link, and the link should go back to the root index.md file that Sugi and Clopedia are adding.

File: y/index.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+List of artists starting with Y
```
Comment (ndorfin) [inline y/index.md]: This needs to turn into a title heading.

**Review by ndorfin — CHANGES_REQUESTED** [review]
This looks great so far, please make the following changes:

**Comment by moro-patana** [comment]
Hey Shaun,
We have done it, please have a look.
Thank you.

**Pull request closed without merging**